src/Plug.vala

```diff
@@ -197,12 +197,22 @@ public class About.Plug : Switchboard.Plug {
             }
         });
 
-        // Bug button
         var bug_button = new Gtk.Button.with_label (_("Report a Problem"));
         bug_button.clicked.connect (() => {
-            var issue_dialog = new IssueDialog ();
-            issue_dialog.transient_for = (Gtk.Window) main_grid.get_toplevel ();
-            issue_dialog.run ();
+            var appinfo = new GLib.DesktopAppInfo  ("io.elementary.feedback.desktop");
```

Comment:
```suggestion
            var appinfo = new GLib.DesktopAppInfo ("io.elementary.feedback.desktop");
```

This might be a nitpicking, but this is double-spaced.